.github/workflows/ci.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh -e
+
+export DD_APPSEC_WAF_TIMEOUT=1m
+export DD_APPSEC_WAF_LOG_FILTER="@ waf[.]cpp:"
+
+# Read all go env and os variables as shell variables
+# shellcheck disable=SC2046
+eval $(go env)
+
+case $GOVERSION in
+    *1.20*|*1.19* )
+        export GODEBUG=cgocheck=2
+        ;;
+    *)
+        export GOEXPERIMENT=cgocheck2
+esac
```

Comment:
This changes the behaviour so much that test must be also run without them being set up.

Here is the simple loop I did in my dd-trace-go CI PR https://github.com/DataDog/dd-trace-go/pull/2541/files#diff-dec1b363c4670835b0b6e96b3cebe6711e688d0dd14701a2cffaf6e6d1f60480R114-R124

Reply:
done